service/service.hpp

```diff
@@ -220,25 +263,40 @@ namespace raven
         DLOG_F(INFO, "cfg.id: %lu", cfg.id.value());
         DLOG_F(INFO, "settings_to_update: %s", cfg.settings_to_update.dump().c_str());
         if (!config_clients_registry_.at(sock.fileno()).has_loaded(cfg.id)) {
-            prepare_answer(sock, request_state::unknown_id);
-            return ;
+            send_answer(sock, request_state::unknown_id);
+            return;
         }
+
         auto db_id = config_clients_registry_.at(sock.fileno()).get_db_id_from(cfg.id);
-        auto state = db_.settings_update(cfg, db_id);
-        prepare_answer(sock, state);
-        if (state != request_state::success)
+
+        auto config_json_data = db_.get_config(db_id);
+        if (db_.fail()) {
+            send_answer(sock, request_state::db_error);
             return ;
+        }
+
+        for (auto &[key, value] : cfg.settings_to_update.items()) {
+            config_json_data["SETTINGS"][key] = value;
```

Comment:
I thought the convention was to avoid hardcoded strings and put them into constants (not sure but it seemed that way)?